tests/specs/install/install_add_dev/package.json.out

```diff
@@ -1,5 +1,3 @@
 {
-  "devDependencies": {
-    "@denotest/esm-basic": "^1.0.0"
-  }
+  "devDependencies": { "@denotest/esm-basic": "^1.0.0" }
```

Comment:
The original file was like this, so since there wasn't any changes it doesn't modify the file.

Reply:
oops, that's an accident. I must have forgotten `"tempdir": true` the first time I ran the test 😅.

could you change `package.json` to just be `{}`?

Reply:
Oh, you're right. I'll change them.